### ISSUE TITLE
Tooltip closes when hovering over <use>d rects from SVG <symbol> in Firefox & Edge

### ISSUE THREAD

**Issue by val-** [twbs/bootstrap#19670]
Bootstrap tooltips work well with inline SVG images.
But I have a trouble with tooltip on SVG symbol image in fiefox.
At first sight it work. But when user move mouse over image tooltip blinking.
I can reproduce this bug in Firefox 45 on Ububntu 14.04 and Firefox 45 on Windows Vista.

Here very simple example http://codepen.io/webkoder/pen/zqENyo

**Comment by twbs-lmvtfy** [comment]
Hi @val-!

You appear to have posted a live example (http://codepen.io/webkoder/pen/zqENyo.html), which is always a good first step. However, according to [the HTML5 validator](http://html5.validator.nu), **your example has some validation errors**, which might potentially be causing your issue:
* line 33, column 3 thru column 8: End tag ```` svg ```` did not match the name of the current open element (```` use ````).

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by val-** [comment]
**twbs-lmvtfy**, thank you. I close `use` tag. Now HTML valid.

**Comment by cvrebert** [comment]
Reposting to trigger re-validation: http://codepen.io/webkoder/pen/zqENyo

**Comment by cvrebert** [comment]
Seems as if Firefox doesn't consider the boxes to be descendants of the label for the purposes of `mouseenter`/`mouseleave`.

**Comment by cvrebert** [comment]
Some work on a debugging pen: http://codepen.io/cvrebert/pen/RajVGb

**Comment by cvrebert** [comment]
Interestingly, this seems to also reproduce in Microsoft Edge 13.

**Comment by cvrebert** [comment]
Filed Firefox bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1278000

**Comment by cvrebert** [comment]
Filed Edge bug: https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/7787318/

**Comment by cvrebert** [comment]
@XhmikosR @hnrch02 Thoughts on whether we could/should attempt a workaround?

**Comment by XhmikosR** [comment]
So this only works right with Chrome?

I'm wondering how exactly we could work around this without too many changes.

**Comment by cvrebert** [comment]
It also works correctly in Safari.
(And given that Edge is largely trying to be compatible with WebKit, it's not unlikely that they might be open to harmonizing with the WebKit/Blink behavior.)

**Comment by hnrch02** [comment]
@cvrebert How do you reckon we would work around this? I guess we could try not immediately hiding the tooltip on mouseleave. Not sure if it's worth it though.

**Comment by cvrebert** [comment]
Yeah, I guess something like "on `mouseleave`, if `relatedTarget` is a descendant of the target, ignore the `mouseleave`"

> Not sure if it's worth it though.

Yeah, I'm likewise ambivalent about this.

**Comment by cvrebert** [comment]
@XhmikosR @hnrch02 Should we deem this WontFix?

**Comment by hnrch02** [comment]
Yeah, as a browser bug.

**Comment by XhmikosR** [comment]
Agreed.

**Comment by cvrebert** [comment]
Closing as WontFix per the consensus then.
Upstream browser bugs have already been filed.
